package.json

```diff
@@ -198,6 +207,8 @@
       "text-summary"
     ],
     "moduleNameMapper": {
+      "^mattermost-redux/test/(.*)$": "<rootDir>/packages/mattermost-redux/test/$1",
+      "^mattermost-redux/(.*)$": "<rootDir>/packages/mattermost-redux/src/$1",
```

Comment:
This is used to rewrite import statements when running in Jest. The test files need their own alias since they don't live in the src folder.